### PR TITLE
Add Elixir & Erlang version to sidebar

### LINKED ISSUE #1380: Add the version info in the left panel
Hi, I think adding the version info for this tutorial in the left panel is a good idea, doesn't it?

When I read this tutorial, I'm not sure which version of Elixir this tutorial teaches.

Maybe we can provide a tutorial list for the different versions of Elixir.

### PR DESCRIPTION
**Pull request by doomspork**
Closes #1380 

I'd like feedback on where this should go, I'll bug @burden for help styling it once we can agree on placement.

### CODE REVIEW

File: _includes/section-sidebar-menu.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <h2>Menu</h2>
     <p>{{ locale.project.description }}</p>
   </header>
+  <span>Elixir {{site.elixir.version}} Erlang/OTP {{site.erlang.otp}} [erts-{{site.erlang.erts}}]</span>
```
Comment (nscyclone) [inline _includes/section-sidebar-menu.html]: I don't see the OTP version on [the screenshot](https://github.com/elixirschool/elixirschool/pull/1409#issuecomment-384401062). Most likely the reason is case-sensitivity of the config keys. Shouldn't it be `{{site.erlang.OTP}}` instead?

Comment (doomspork) [reply _includes/section-sidebar-menu.html]: image

File: _includes/section-sidebar-menu.html [hunk 2]
```diff
@@ -5,6 +5,7 @@
     <h2>Menu</h2>
     <p>{{ locale.project.description }}</p>
   </header>
+  <span>Elixir {{site.elixir.version}} Erlang/OTP {{site.erlang.OTP}} [erts-{{site.erlang.erts}}]</span>
```
Comment (eksperimental) [inline _includes/section-sidebar-menu.html]: We need something to separate Elixir version from Erlang/OTP other than just a space.
maybe a `-`

Comment (doomspork) [reply _includes/section-sidebar-menu.html]: image

**Pull request merged**